### PR TITLE
Process bean security

### PR DESCRIPTION
**Pull request by ivo-ritense**

### CODE REVIEW

File: core/src/main/kotlin/com/ritense/valtimo/ValtimoExpressionManager.kt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    override fun createValueExpression(expression: String?): ValueExpression {
+        ensureInitialized()
+        val expression = expressionFactory.createValueExpression(parsingElContext, expression, Any::class.java)
+        return ValtimoValueExpression(expression)
+    }
+}
+
+class ValtimoValueExpression(
+    private val expression: ValueExpression
+): ValueExpression() {
```
Comment (Klaas-Ritense) [inline core/src/main/kotlin/com/ritense/valtimo/ValtimoExpressionManager.kt]:
```suggestion
): ValueExpression by expression {
```

File: core/src/test/kotlin/com/ritense/valtimo/camunda/authorization/CamundaBeanAuthorizationIT.kt [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.ritense.valtimo.BaseIntegrationTest
+import org.junit.jupiter.api.Test
+import java.util.UUID
+import javax.transaction.Transactional
+
+@Transactional
+class CamundaBeanAuthorizationIT: BaseIntegrationTest() {
+
+    @Test
+    fun `should be able to execute process bean without authorizing`() {
```
Comment (theo-ritense) [inline core/src/test/kotlin/com/ritense/valtimo/camunda/authorization/CamundaBeanAuthorizationIT.kt]: This currently only tests the service task. The story describes some others. Shouldn't there be test-cases for those as well?

File: core/src/main/kotlin/com/ritense/valtimo/ValtimoExpressionManager.kt (hunk 1 above)
Comment (ivo-ritense) [reply core/src/main/kotlin/com/ritense/valtimo/ValtimoExpressionManager.kt]: "Only interfaces can be delegated to"

File: core/build.gradle [hunk 3]
```diff
@@ -121,6 +121,7 @@ dependencies {
     testImplementation "org.camunda.community.mockito:camunda-platform-7-mockito:${camundaMockitoVersion}"
     testImplementation "org.springframework.security:spring-security-test"
     testImplementation "org.junit.jupiter:junit-jupiter-engine"
+    testImplementation "org.codehaus.groovy:groovy-all:3.0.20"
```
Comment (Klaas-Ritense) [inline core/build.gradle]: Shouln't the version be in the `gradle.properties` file?

File: core/src/test/kotlin/com/ritense/valtimo/camunda/authorization/CamundaBeanAuthorizationIT.kt (hunk 2 above)
Comment (ivo-ritense) [reply core/src/test/kotlin/com/ritense/valtimo/camunda/authorization/CamundaBeanAuthorizationIT.kt]: I added test cases for all the examples named in the story. The expectation for the current solution is that it should also work for other applications that you can configure in bpmn.

File: core/build.gradle (hunk 3 above)
Comment (ivo-ritense) [reply core/build.gradle]: Moved the version to `gradle.properties` and updated to the latest version

**Review by Klaas-Ritense — APPROVED** [review]

**Pull request merged**